### PR TITLE
initiate pull request

### PR DESCRIPTION
**Pull request by Ricky-Kynd**

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+				<p>This is a learning exercise as well as a demonstration. You retain full copyright on your graphics (with limited exceptions, see <a href="http://www.mezzoblue.com/zengarden/submit/guidelines/">submission guidelines</a>), but we ask you release your <abbr title="Cascading Style Sheets">CSS</abbr> under a Creative Commons license identical to the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Zen Garden's license information.">one on this site</a> so that others may learn from your work.
+				</p>
+				<p role="contentinfo">By <a href="http://www.mezzoblue.com/">Dave Shea</a>. Bandwidth graciously donated by <a href="http://www.mediatemple.net/">mediatemple</a>. Now available: <a href="http://www.amazon.com/exec/obidos/ASIN/0321303474/mezzoblue-20/">Zen Garden, the book</a>.</p>
+		</div>
+
+		<footer>
+		  		<a href="http://validator.w3.org/check/referer" title="Check the validity of this site&#8217;s HTML" class="zen-validate-html">HTML</a>
+		    	<a href="http://jigsaw.w3.org/css-validator/check/referer" title="Check the validity of this site&#8217;s CSS" class="zen-validate-css">CSS</a>
```
Comment (ghost) [inline pristine-zen-garden.html]: This link isn't indented the same as the others.

File: pristine-zen-garden.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+<body id="css-zen-garden">
+	<div class="page-wrapper">
+		<section class="intro" id="zen-intro">
+			<header role="banner">
+				<h1>CSS Zen Garden</h1>
+				<h2>The Beauty of <abbr title="Cascading Style Sheets">CSS</abbr> Design</h2>
+			</header>
+
+			<div class="summary" id="zen-summary" role="article">
+					<p>A demonstration of what can be accomplished through <abbr title="Cascading Style Sheets">CSS</abbr>-based design. Select any style sheet from the list to load it into this page.</p>
```
Comment (ghost) [inline pristine-zen-garden.html]: The resource suggests that content should be indented separately from the code tags.

File: pristine-zen-garden.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+				<p>Download the sample <a href="/examples/index" title="This page's source HTML code, not to be modified.">HTML</a> and <a href="/examples/style.css" title="This page's sample CSS, the file you may modify.">CSS</a> to work on a copy locally. Once you have completed your masterpiece (and please, don&#8217;t submit half-finished work) upload your <abbr title="Cascading Style Sheets">CSS</abbr> file to a web server under your control. <a href="http://www.mezzoblue.com/zengarden/submit/" title="Use the contact form to send us your CSS file">Send us a link</a> to an archive of that file and all associated assets, and if we choose to use it we will download it and place it on our server.</p>
+		</div>
+
+		<div class="benefits" id="zen-benefits" role="article">
+			<h3>Benefits</h3>
+				<p>Why participate? For recognition, inspiration, and a resource we can all refer to showing people how amazing <abbr title="Cascading Style Sheets">CSS</abbr> really can be. This site serves as equal parts inspiration for those working on the web today, learning tool for those who will be tomorrow, and gallery of future techniques we can all look forward to.
+				</p>
```
Comment (ghost) [inline pristine-zen-garden.html]: I think you need some consistency in your styling. Most of your paragraph tags open and close on the same line except here and on line 61

File: pristine-zen-garden.html [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+				<p>We ask that you submit original artwork. Please respect copyright laws. Please keep objectionable material to a minimum, and try to incorporate unique and interesting visual themes to your work. We&#8217;re well past the point of needing another garden-related design.</p>
+				<p>This is a learning exercise as well as a demonstration. You retain full copyright on your graphics (with limited exceptions, see <a href="http://www.mezzoblue.com/zengarden/submit/guidelines/">submission guidelines</a>), but we ask you release your <abbr title="Cascading Style Sheets">CSS</abbr> under a Creative Commons license identical to the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Zen Garden's license information.">one on this site</a> so that others may learn from your work.
```
Comment (ghost) [inline pristine-zen-garden.html]: See note above.

File: pristine-zen-garden.html [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+			</div>
+
+			<div class="design-archives" id="design-archives">
+				<h3 class="archives">Archives:</h3>
+					<nav role="navigation">
+						<ul>
+							<li class="next">
+								<a href="/214/page3">Next Designs <span class="indicator">&rsaquo;</span></a>
+							</li>
+
```
Comment (ghost) [inline pristine-zen-garden.html]: I would omit the linespace between each list item as it makes the code really long. You've left the linespace out above, but have put them in here and in the resources div list.

**Comment by ghost** [comment]
Hey @Ricky-Kynd I think you've used double tabs throughout your document for your indentation instead of 2 spaces or 4 spaces.

**Comment by Ricky-Kynd** [comment]
@zippo-yap Shouldn't have added you... lol

File: pristine-zen-garden.html (hunk 5 above)
Comment (Ricky-Kynd) [inline pristine-zen-garden.html]: I was going to put comments in between them.

File: pristine-zen-garden.html (hunk 2 above)
Comment (Ricky-Kynd) [inline pristine-zen-garden.html]: Could you send me a link to this resource?

File: pristine-zen-garden.html (hunk 3 above)
Comment (Ricky-Kynd) [inline pristine-zen-garden.html]: Hater's go'n hate.

**Comment by Ricky-Kynd** [comment]
@zippo-yap This does not look like what's in my ide. My tab indentation spaces are set at 4 tabs as oppose to 2.

File: pristine-zen-garden.html (hunk 2 above)
Comment (ghost) [inline pristine-zen-garden.html]: https://github.com/dev-academy-programme/curriculum/tree/master/resources/html-formatting-ARTICLE :)

**Pull request merged**